v5/api/c/gps.rst

```diff
@@ -0,0 +1,681 @@
+ Parameters
+=============== =================================================================================================================
+port             The V5 GPS port number from (1-21)
+=============== =================================================================================================================
+
+**Returns:** The raw gyroscope values. If the operation failed, all the structure's members are filled with ``PROS_ERR_F`` and 
+``errno`` is set.
+
+----
+
+gps_get_accel
+----------------
+
+Get the GPS's raw accelerometer values.
+
+This function uses the following values of ``errno`` when an error state is reached:
+
+- ``ENXIO`` - The given value is not within the range of V5 ports (1-21).
+- ``ENODEV`` - The port cannot be configured as a GPS.
+- ``EAGAIN``` - The sensor is still calibrating.
+
+.. tabs ::
+   .. tab :: Prototype
+      .. highlight:: c
+      ::
+
+         gps_accel_s_t gps_get_accel(uint8_t port)
+
+   .. tab :: Example
+      .. highlight:: c
+      ::
+        #define GPS_PORT 1
+
+        void opcontrol() {
+            struct gps_accel_s_t accel;
+
+            while (true) {
+                accel = gps_get_accel(GPS_PORT);
+                screen_print(TEXT_MEDIUM, 1, "accleration- x: %3f, y: %3f, z: %3f", accel.x, accel.y, accel.z);
+            }
+        }
+
+=============== =================================================================================================================
+ Parameters
+=============== =================================================================================================================
+port             The V5 GPS port number from (1-21)
+=============== =================================================================================================================
+
+**Returns:** The raw accelerometer values. If the operation failed, all the structure's members are filled with ``PROS_ERR_F`` and 
+``errno`` is set.
+
+----
+
+Macros
+======
+
+None.
+
+Enumerated Values
+=================
+
+None.
+
+Structures
+==========
+
+gps_status_s_t
+--------------
+
+::
+
+  typedef struct __attribute__((__packed__)) gps_status_s {
+	double x;
+	double y;
+	double pitch;
+	double roll;
+	double yaw;
+  } gps_status_s_t;
+
+================================== =====================================================================================
+ Value
+================================== =====================================================================================
+x                                   X Position (meters)
+y                                   Y Position (meters)
+pitch                               Percieved Pitch based on GPS and IMU
+roll                                Percieved Roll based on GPS and IMU
+yaw                                 Percieved Yaw based on GPS and IMU
+================================== =====================================================================================
+
+---
```

Comment:
I think this is causing some weird lines in the produced files, applies to the C++ api as well.

Reply:
It seems this was intentional, but is inconsistent throughout the docs whether it be a short line or a long line. I'm gonna make the GPS's consistent at the very least by making them long lines.